src/main/scala/org/broadinstitute/hail/driver/ImportPlink.scala

```diff
@@ -30,7 +30,7 @@ object ImportPlink extends Command {
 
     @Args4jOption(required = false, name = "-d", aliases = Array("--delimiter"),
       usage = ".fam file field delimiter regex")
-    var famDelimiter: String = "\\t"
+    var famDelimiter: String = "\\\\s+"
```

Comment:
I think those are invalid for plink import.  I'd rather that be the special case than whitespace-delimited files